querydsl-core/src/main/java/com/querydsl/core/types/dsl/PathBuilder.java

```diff
@@ -44,9 +44,9 @@
 
     private static final long serialVersionUID = -1666357914232685088L;
 
-    private final Map<String, PathBuilder<?>> properties = Maps.newHashMap();
+    private final ConcurrentHashMap<String, PathBuilder<?>> properties = new ConcurrentHashMap<String, PathBuilder<?>>();
```

Comment:
`Maps.newConcurrentHashMap()` returns a different implementation, `ConcurrentHashMap` is what's used in the superclasses.

The declaration change is deliberate. I'm forcing the type, as we *always* want a `ConcurrentHashMap`.